### ISSUE TITLE
Is it possible to generate a strategy for Oracle?

### ISSUE THREAD

**Issue by gansai** [epiresdasilva/cte-multi-table-bulk-id-stategy#4]
As I have seen, your code uses CTE and this strategy can be used for Postgres.
But, can we make a generic strategy, which, is irrespective of whichever database, it can ultimately avoid creating temporary tables.

As, I am facing some permission issue with Oracle and not able to create temporary tables.

**Comment by epiresdasilva** [comment]
Of course, yes. I think it's a very good idea. Can you contribute with this development?

**Comment by gansai** [comment]
would love to. I shall update you once I come up with some design/idea on how to solve this.
